cmd/kubeadm/app/util/system/types_windows.go

```diff
@@ -38,7 +38,7 @@ var DefaultSysSpec = SysSpec{
 	Cgroups: []string{},
 	RuntimeSpec: RuntimeSpec{
 		DockerSpec: &DockerSpec{
-			Version:     []string{`18\.06\..*`}, //Requires [18.06] or later
+			Version:     []string{`18\.09\..*`}, //Requires [18.09] or later
```

Comment:
let's make this one:
`18\.0[6,9]\..*`

you also need to modify:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/system/docker_validator_test.go#L31

and make -> `18\.06\..*` -> `18\.0[6,9]\..*`